langwatch/src/server/api/routers/traces.ts

```diff
@@ -524,11 +547,15 @@ const getAllForProject = async (
     ...(usePivotIndex ? [{ terms: { trace_id: traceIds } }] : []),
   ];
 
-  const canSeeCosts = await backendHasTeamProjectPermission(
-    ctx,
-    input,
-    TeamRoleGroup.COST_VIEW
-  );
+  let canSeeCosts = false;
+  if (ctx.prisma) {
```

Comment:
feels like a hack since this is actually not optional, you had to `// @ts-ignore` the called, can you make it explicitly optional on the function signature and remove the `// @ts-ignore` from triggers.ts?